general/mem_manager.hpp

```diff
@@ -59,7 +95,7 @@ MemoryType GetMemoryType(MemoryClass mc);
     Currently, the operation is defined as a*b := max(a,b) where the max
     operation is based on the enumeration ordering:
 
-        HOST < HOST_32 < HOST_64 < CUDA < CUDA_UVM. */
+    HOST < HOST_32 < HOST_64 < DEVICE < MANAGED. */
```

Comment:
This comment also needs to be updated.

Reply:
These are the `MemoryClass`: we have all the five of them here.

Reply:
You are right, my mistake.